### DISCUSSION TITLE
Semi-ATE installation

### DISCUSSION THREAD

**Discussion by nerohmot** [Semi-ATE/Semi-ATE#225] [Q&A]
1. make a new environment (`_Semi-ATE_`) with a python 3.9 : `mamba create -n _Semi-ATE_ python=3.9 spyder=5.3.0` 
2. Activate the new environment : `conda activate _Semi-ATE_`
3. go to the (checked out repo of `Semi-ATE`) : `cd ~/Repos/Semi-ATE/Semi-ATE`
4. change to the script directory : `cd scripts`
5. execute the command : `python package_tool.py --change-env cicd`
6. start spyder by typing : `spyder`

**Comment by Zlin526F** [comment]
A more detailed installation for User without a clone from semi-ate:
  1. install maxiconda from [https://www.maxiconda.org](https://www.maxiconda.org/), 
     - don't install maxiconda as an administrator, otherwise you need admin rights for each package that is to be installed later. This is not practical. And you should install maxiconda only for you, not for all users.
     - Maxiconda is not running together with Anaconda, you  have to  deinstall Anaconda first! 
      - if c:\\users\\<your username>\\.condarc exist, than remove this file.
  2. install [mosquitto](https://mosquitto.org/download/)
  3. call a maxiconda powershell (for windows user) or a shell for ubuntu/linux user
  4. cd  ~
  5. mkdir repos
  6. cd repos
  7. mkdir Semi-ATE
  8. cd Semi-ATE
  9. git clone https://github.com/Semi-ATE/Semi-ATE.git
now you can start with the above installation
